spec/models/review_spec.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+  let!(:review) { create :review }
+  it { should validate_presence_of(:rating) }
+  it { should_not allow_value('').for(:rating) }
+  it { should_not allow_value(nil).for(:rating) }
```

Comment:
I think that there should be better method for checking `''` or `nil` - check it in rspec doc